components/AudioPlayer.vue

```diff
@@ -0,0 +1,82 @@
+<script setup>
+import VPersistentPlayer from '@nypublicradio/nypr-design-system-vue3/v2/src/components/VPersistentPlayer.vue'
+import { useCurrentEpisode } from '~/composables/states'
+import { toastGlobalConfig } from '~/utilities/helpers'
+import { createToast } from 'mosha-vue-toastify'
+// had to install howler.js locally and add this import to stop it from breaking the build
```

Comment:
I have NOOOOO idea, but I had to install Howler locally and call the imports here for the error to go away... which was basically saying that it could not find the export "Howl" in howler

Reply:
Hmm whatever works is good for now… We can always go back and try to figure it out